py/strato/common/log/configurelogging.py

```diff
@@ -72,6 +72,11 @@ def addFileHandler(name, path):
     logging.info("Logging started for %s" % name)
 
 
+def changeHandlerLogLevelbyHandlerType(logger, logLevel, handlerType=None):
+    """ if not specifying handlers type , change log logleve to all """
```

Comment:
why do u set handlerType to None? It's not acceptable value.
Just add: assert handlerType is not None and dont run the loop.